### PR TITLE
remove std_msgs

### PR DESCRIPTION
**Pull request by taikitanaka3**
remove std_msgs and add autoware debug msgs

### CODE REVIEW

File: control/velocity_controller/src/velocity_controller_core.cpp [hunk 1]
```diff
@@ -641,7 +650,8 @@ bool VelocityController::isEmergencyState(int closest, double target_vel) const
   return false;
 }
 
-bool VelocityController::isValidTrajectory(const autoware_planning_msgs::msg::Trajectory & traj) const
+bool VelocityController::isValidTrajectory(const autoware_planning_msgs::msg::Trajectory & traj)
+const
```
Comment (wep21) [inline control/velocity_controller/src/velocity_controller_core.cpp:654]: It looks a little bit wierd.  I think it's better as below.
```
bool VelocityController::isValidTrajectory(
  const autoware_planning_msgs::msg::Trajectory & traj) const
```

File: control/velocity_controller/src/velocity_controller_mathutils.cpp [hunk 2]
```diff
@@ -69,7 +73,8 @@ bool calcClosestWithThr(
     double yaw_pose = tf2::getYaw(pose.orientation);
     double yaw_ref = tf2::getYaw(trajectory.points.at(i).pose.orientation);
     double yaw_diff = normalizeEulerAngle(yaw_pose - yaw_ref);
-    // printf("i = %d, yaw_pose = %f, yaw_ref = %f, yaw_diff = %f\n", i, yaw_pose, yaw_ref, yaw_diff);
+    // printf("i = %d, yaw_pose = %f, yaw_ref = %f, yaw_diff
+    // = %f\n", i, yaw_pose, yaw_ref, yaw_diff);
```
Comment (wep21) [inline control/velocity_controller/src/velocity_controller_mathutils.cpp:77]: It looks a little bit wierd. I think it's better as below.
```
// printf(
//   "i = %d, yaw_pose = %f, yaw_ref = %f, yaw_diff = %f\n", i, yaw_pose, yaw_ref, yaw_diff);
```

File: control/velocity_controller/src/velocity_controller_core.cpp (hunk 1 above)
Comment (taikitanaka3) [reply control/velocity_controller/src/velocity_controller_core.cpp:654]: this is cause by uncursify i will fix this

**Review by wep21 — APPROVED** [review]
LGTM

**Pull request merged**